agent/src/beerocks/slave/beerocks_slave_main.cpp

```diff
@@ -149,8 +149,20 @@ static void fill_son_slave_config(beerocks::config_file::sConfigSlave &beerocks_
     son_slave_conf.hostap_iface = beerocks_slave_conf.hostap_iface[slave_num];
     son_slave_conf.hostap_ant_gain =
         beerocks::string_utils::stoi(beerocks_slave_conf.hostap_ant_gain[slave_num]);
-    son_slave_conf.radio_identifier        = beerocks_slave_conf.radio_identifier[slave_num];
-    son_slave_conf.backhaul_wireless_iface = beerocks_slave_conf.sta_iface[slave_num];
+    son_slave_conf.radio_identifier = beerocks_slave_conf.radio_identifier[slave_num];
+    // read the sta_iface from bpl and verify it is available
+    char sta_iface_str[BPL_IFNAME_LEN];
```

Comment:
Please move this to a function to keep the main small. The function should be generic if possible since some of the other parameters here will also move to BPL as well.

Reply:
ok - moved to a separate static function